sdk/core/Azure.Core/src/Shared/NullableAttributes.cs

```diff
@@ -105,7 +109,8 @@ sealed class NotNullIfNotNullAttribute : Attribute
 #else
     public
 #endif
-        sealed class DoesNotReturnAttribute : Attribute { }
+        sealed class DoesNotReturnAttribute : Attribute
```

Comment:
@chidozieononiwu can you please also revert these occurrences of unfortunate formatting?

Reply:
Sorry I missed those. Its's reverted now.